GETTING_STARTED.md

```diff
@@ -0,0 +1,39 @@
+# getting_started
+
+Follow this guide to setup [Parrot Security](https://www.parrotsec.org/) with Docker on Oracle VirtualBox. Similar steps can be taken to setup [Kali Linux](https://www.kali.org/) instead.
+
+## Downloads
+
+- Download [Parrot Security OVA](https://www.parrotsec.org/download/) virtual machine image and load it on VirtualBox
+
+- Download and install [Oracle VirtualBox](https://www.virtualbox.org/wiki/Downloads)
+
+## Virtual Machine Setup
+
+- Launch VirtualBox and make a **New** virtual machine. Choose a name and select Linux in *Type* and Debian (64-bit) in *Version*.
+
+- Assign resources according with [Parrot Security system requirements](https://docs.parrotlinux.org/info/system-requirements/) (e.g at least 2 GB of RAM and 40 GB hard drive)
+
+- Create a Dynamically allocated Virtual Hard Disk and select VHD.
+
+- Once created, use the Settings -> System tab to make sure *Optical* is first in Boot Order
+
+- Also in Settings, but in the Storage tab, select *Add Optical Drive* (the disk  :cd: icon next to *Controller: IDE*) and select the Parrot Security image you downloaded previously.
+
+## Parrot Security Install
+- Launch your virtual machine and follow the installer's instructions. The *Install with GTK GUI* option is recommended for ease of use. You should be ready to boot Parrot Security OS afterwards.
+
+## Docker Installation on Parrot Security  
+
+- Inside your Parrot OS, [Install Docker Engine on Debian
+](https://docs.docker.com/engine/install/debian/) with **one important change**. Add the repository to sources list by:
```

Comment:
Can you place this as a reference link and put the link at the bottom of the page. E.g. `[Install Docker..... Debian][docker-debian]` and at the bottom of the page `[docker-debian]: paste-link-here`